engine/src/main/java/org/terasology/engine/rendering/assets/skeletalmesh/SkeletalMeshDataBuilder.java

```diff
@@ -53,33 +53,33 @@ public SkeletalMeshDataBuilder addWeights(List<BoneWeight> weights) {
         return this;
     }
 
-    public SkeletalMeshDataBuilder addMesh(Bone bone, MeshBuilder builder) {
-        return addMesh(bone, builder.getMeshData());
-    }
-
-    public SkeletalMeshDataBuilder addMesh(Bone bone, StandardMeshData data) {
-        TFloatList meshVertices = data.getVertices();
-        TIntList meshIndices = data.getIndices();
-        TFloatList texCoord0 = data.uv0;
-        int weightsStart = weights.size();
-        addBone(bone);
-        for (int i = 0; i < meshVertices.size() / 3; i++) {
-            float x = meshVertices.get(i * 3);
-            float y = meshVertices.get(i * 3 + 1);
-            float z = meshVertices.get(i * 3 + 2);
-            Vector3f pos = new Vector3f(x, y, z);
-            BoneWeight weight = new BoneWeight(new float[]{1}, new int[]{bone.getIndex()});
-            // TODO Copy mesh normals
-            vertices.add(pos);
-            weights.add(weight);
-            uvs.add(new Vector2f(texCoord0.get(i * 2), texCoord0.get(i * 2 + 1)));
-        }
-
-        for (int i = 0; i < meshIndices.size(); i++) {
-            indices.add(meshIndices.get(i) + weightsStart);
-        }
-        return this;
-    }
+//    public SkeletalMeshDataBuilder addMesh(Bone bone, MeshBuilder builder) {
+//        return addMesh(bone, builder.getMeshData());
+//    }
+//
+//    public SkeletalMeshDataBuilder addMesh(Bone bone, StandardMeshData data) {
+//        TFloatList meshVertices = data.getVertices();
+//        TIntList meshIndices = data.getIndices();
+//        TFloatList texCoord0 = data.uv0;
+//        int weightsStart = weights.size();
+//        addBone(bone);
+//        for (int i = 0; i < meshVertices.size() / 3; i++) {
+//            float x = meshVertices.get(i * 3);
+//            float y = meshVertices.get(i * 3 + 1);
+//            float z = meshVertices.get(i * 3 + 2);
+//            Vector3f pos = new Vector3f(x, y, z);
+//            BoneWeight weight = new BoneWeight(new float[]{1}, new int[]{bone.getIndex()});
+//            // TODO Copy mesh normals
+//            vertices.add(pos);
+//            weights.add(weight);
+//            uvs.add(new Vector2f(texCoord0.get(i * 2), texCoord0.get(i * 2 + 1)));
+//        }
+//
+//        for (int i = 0; i < meshIndices.size(); i++) {
+//            indices.add(meshIndices.get(i) + weightsStart);
+//        }
+//        return this;
+//    }
```

Comment:
Why is this commented out? Is it no longer needed? If so, just remove it.

Reply:
the concept is you can build a static mesh and attach it to an existing bone :? kind of strange and niche and apparently unused so I dropped it. I want to refactor this to use a vertex shader to instead of constantly updating vertex data on the CPU.